src/features/dataset/DatasetNameCommitModal.tsx

```diff
@@ -16,12 +16,12 @@ import { ModalType, selectNavExpanded } from "../app/state/appState"
 import { QriRef, newQriRef } from "../../qri/ref"
 import { loadDatasetCommits } from "../commits/state/commitActions"
 
-export interface DatasetHeaderLayoutProps {
+export interface DatasetNameCommitModalProps {
```

Comment:
Is this for a future flow?

Reply:
This is actually that modal that we had in place to edit the title with a commit button, but we've decided to not use it for now.  The component is still around if we choose to use it again.